packages/html/src/menu/menu-scrollable-wrapper.spec.tsx

```diff
@@ -0,0 +1,50 @@
+import { classNames } from '../misc';
+
+export const MENU_SCROLLABLE_WRAPPER_CLASS = `k-menu-scroll-wrapper`;
+
+const states = [];
+
+const options = {};
+
+export type KendoMenuScrollableWrapperProps = {
+    orientation?: string;
+    style?: React.CSSProperties;
```

Comment:
Do we need this _style_ prop? We usually set styles through _style attribute_ directly